app/assets/stylesheets/uservoice.css.erb

```diff
@@ -1,25 +0,0 @@
-/* =USERVOICE ICON DEFINE
---------------------------------------------------------*/
-
-.unauthenticated .uv-icon {
-	display: none;
-}
-
-div.uv-icon.uv-bottom-left {
-	background-image: url(<%= asset_data_uri 'feedback_sprite.png' %>);
```

Comment:
we could leave this stuff in and make it a link... change the classnames or not bother